app/controllers/trips_controller.rb

```diff
@@ -0,0 +1,52 @@
+class TripsController < ApplicationController
+  rescue_from AddressNotFound, with: :render_address_not_found
+
+  def show
+    render json: TripSerializer.new(trip), status: 200
```

Comment:
Is it necessary to return `status: 200` here? Isn't it default?

Reply:
Dominik gave me a suggestion that at some point sb in project can change default status code so I should declare it in each render